doc/source/getting-started/installation.rst

```diff
@@ -7,15 +7,15 @@ Operating systems support
 In the table below we summarize the status of *pre-compiled binaries
 distributed with pypi* for the packages listed above.
 
-+------------------+------+---------+------------+
-| Operating System | qibo | qibojit | tensorflow |
-+==================+======+=========+============+
-| Linux x86        | Yes  | Yes     | Yes        |
-+------------------+------+---------+------------+
-| MacOS >= 10.15   | Yes  | Yes     | Yes        |
-+------------------+------+---------+------------+
-| Windows          | Yes  | Yes     | Yes        |
-+------------------+------+---------+------------+
++------------------+------+---------+-----------+---------+
+| Operating System | qibo | qibojit |Tensorflow | Pytorch |
++==================+======+=========+===========+=========+
+| Linux x86        | Yes  | Yes     | Yes       | Yes     |
++------------------+------+---------+-----------+---------+
+| MacOS >= 10.15   | Yes  | Yes     | Yes       | Yes     |
++------------------+------+---------+-----------+---------+
+| Windows          | Yes  | Yes     | Yes       | Yes     |
++------------------+------+---------+-----------+---------+
```

Comment:
Should this be updated to contemplate the CPU / GPU subdivision? Both `qibojit` and `qiboml` have backends that work on both.

Reply:
I don't know, since this is already shown in the backends diagram.